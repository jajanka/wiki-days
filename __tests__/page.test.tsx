import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '@/app/page';

jest.useFakeTimers().setSystemTime(1_739_987_188_173);

describe('Main page', () => {
  it('renders visually correct', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
