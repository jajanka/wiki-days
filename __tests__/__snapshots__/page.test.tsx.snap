// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page renders visually correct 1`] = `
<DocumentFragment>
  <div
    class="title "
  >
    Notable birthdays on this day
  </div>
  <div
    class="container"
  >
    <button
      class="btn btn-full"
    >
      Get Birthdays
    </button>
    <div
      class="date-title"
    >
      Wed Feb 19 2025
    </div>
  </div>
</DocumentFragment>
`;
